src/operator/subgraph/partition_graph.cc

```diff
@@ -484,6 +484,21 @@ void SortEntries(const std::unordered_map<const nnvm::NodeEntry*, size_t>& entry
   std::sort(entries->begin(), entries->end(), entry_cmp);
 }
 
+// Remove duplicate Node Entries from a vector
```

Comment:
could set make it simpler...

`set<int>` s( vec.begin(), vec.end() );
vec.assign( s.begin(), s.end() );`

https://stackoverflow.com/questions/1041620/whats-the-most-efficient-way-to-erase-duplicates-and-sort-a-vector